recipes/mosquitto/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "mosquitto" %}
+{% set version = "2.0.14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://mosquitto.org/files/source/{{ name }}-{{ version }}.tar.gz
+  sha256: d0dde8fdb12caf6e2426b4f28081919a2fce3448773bdb8af0d3cd5fe5776925
+
+build:
+  number: 0
+  skip: True  # [osx]
+  run_exports:
+    - {{ pin_subpackage( 'mosquitto' ) }}
+  ignore_run_exports:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
```

Comment:
```suggestion
```
What libraries are you trying to ignore with this section?

Reply:
😟 was messing around to try to get the warnings to go away 🥴